.pre-commit-config.yaml

```diff
@@ -16,6 +16,15 @@ repos:
     hooks:
       - id: isort
         language_version: python3
+  # Flake8: complexity and style checking
+  # https://flake8.pycqa.org/en/latest/user/using-hooks.html
+  - repo: https://github.com/pycqa/flake8
+    rev: 4.0.1
+    hooks:
+      - id: flake8
+        additional_dependencies: [flake8-docstrings]
+        exclude: (^docs/|^dataprofiler/tests/)
+        language_version: python3.9
```

Comment:
add PR #547 to resolve an issue I found... going forward I think we should just set this value as `python3`